spec/features/review/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "remove a review for a restaurant", %q(
+  As a user I want to remove a review I added for a given restaurant
+) do
+
+  scenario "Authenticated user successfully delete a review they made" do
+    restaurant = FactoryGirl.create(:restaurant)
+    user = FactoryGirl.create(:user)
+    sign_in_as(user)
+
+    review_owned_by_user = FactoryGirl.create(
+      :review, restaurant: restaurant, user: user
+    )
+
+    # user needs to click on delete next to the review that belongs to them
+    visit restaurant_review_path(
+      restaurant, review_owned_by_user, method: :delete
+    )
+
```

Comment:
You can clean up a lot of this code as long as your associations are in place:

```ruby
review = FactoryGirl.create(:review)
user = review.user
restaurant = review.restaurant
```

You can then `sign_in_as(user)` and check if the delete works